waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/web/controllers/auth/x5c/X5cAuthRoutes.kt

```diff
@@ -7,7 +7,7 @@ const val x5cAuthPath = "auth/x5c"
 val x5cAuthTags = listOf("X5C Authentication")
 
 fun Application.x5cAuthRoutes() = webWalletRoute {
-    X5CLoginController().routes("login").invoke(this)
+    X5CLoginController().routes("login")(this)
```

Comment:
and this one also coming from https://github.com/walt-id/waltid-identity/pull/746
what happened with the merge?